crates/ursa-utils/src/contract_bindings/fleek_reward_binding.rs

```diff
@@ -0,0 +1,11907 @@
+    ///The parsed JSON ABI of the contract.
+    pub static FLEEKREWARD_ABI: ::ethers::contract::Lazy<::ethers::core::abi::Abi> = ::ethers::contract::Lazy::new(||
+    ::ethers::core::utils::__serde_json::from_str(__ABI).expect("ABI is always valid"));
+    #[rustfmt::skip]
+    const __BYTECODE: &[u8] = &[
```

Comment:
Is this formatting intentional?

Reply:
It's not visible in the preview here but essentially each byte is stored in a separate line.

Reply:
Yeah these autogenerated ethers files are kinda ugly thats how it generates. Ive refactored them out in the dynamic-epoch branch. I have to rebase this branch the same way so it will probably be good to still merge this and ill rebase these files out before we go to main

Reply:
no, these bindings are autogenereted. I will fix this for this commit.
I and Dalton have a plan to get rid of these bindings before we merge into the main